GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/PurchaseController.cs

```diff
@@ -319,28 +319,33 @@ private static void UnlockInGameContent(string productId)
             return;
         }
 
-        
+
         // Pop up window
         Debug.LogError("Product ID doesn't match any of exist products.");
     }
 
     public void OnPurchaseFailed(Product product, PurchaseFailureReason failureReason)
     {
-        
+
         // A product purchase attempt did not succeed. Check failureReason for more detail. Consider sharing
         // this reason with the user to guide their troubleshooting actions.
         Debug.Log(
             $"OnPurchaseFailed: FAIL. Product: '{product.definition.storeSpecificId}', PurchaseFailureReason: {failureReason}");
 
         // When purchase fail due to duplicate transaction, and the item is not shown in client side,
         // do restore purchase to fetch the product.
-        // This situation may happen when the user lose internet connection after paying to play store. 
+        // This situation may happen when the user lose internet connection after paying to play store.
         if (failureReason == PurchaseFailureReason.DuplicateTransaction)
         {
             _playStoreExtensions.RestoreTransactions(null);
         }
     }
 
+    public static void confirmSubscriptionPriceChange(string productId)
+    {
+        _playStoreExtensions.ConfirmSubscriptionPriceChange(productId, delegate(bool success) { });
```

Comment:
You need to implement this call back. For the simplest user case, use Debug.log to print it out and add comment saying that devs can choose to make update or record that user has accepted the new price.

Reply:
I see, will do 👍